includes/payment-methods/class-klarna-payment-method.php

```diff
@@ -34,29 +34,69 @@ public function __construct( $token_service ) {
 		$this->countries                    = [ 'US', 'GB', 'AT', 'DE', 'NL', 'BE', 'ES', 'IT', 'IE', 'DK', 'FI', 'NO', 'SE' ];
 		$this->limits_per_currency          = [
 			'USD' => [
-				'min' => 1000,
-				'max' => 500000,
-			], // Represents USD 10 - 5,000.
+				'US' => [
+					'min' => 0,
+					'max' => 1000000,
```

Comment:
Sidenote: if minimum supported PHP version is 7.4, we can start using `1_000_000` to write numbers, or, in this case, even `10_000_00` for better readability.